sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizedForm.cs

```diff
@@ -10,13 +10,15 @@ namespace Azure.AI.FormRecognizer.Models
     /// </summary>
     public class RecognizedForm
     {
-        internal RecognizedForm(PageResult_internal pageResult, IReadOnlyList<ReadResult_internal> readResults)
+        internal RecognizedForm(IReadOnlyList<PageResult_internal> pageResults, IReadOnlyList<ReadResult_internal> readResults, int pageIndex)
         {
+            PageResult_internal pageResult = pageResults[pageIndex];
+
             // Recognized form from a model trained without labels.
             FormType = $"form-{pageResult.ClusterId}";
             PageRange = new FormPageRange(pageResult.Page, pageResult.Page);
-            Fields = ConvertFields(pageResult.Page, pageResult.KeyValuePairs, readResults);
-            Pages = ConvertPages(new List<PageResult_internal>() { pageResult }, readResults);
+            Fields = ConvertUnsupervisedFields(pageResult.Page, pageResult.KeyValuePairs, readResults);
+            Pages = ConvertUnsupervisedPages(pageResult, readResults, pageIndex);
```

Comment:
It would be worth adding a comment here to say we're creating a single page because service has 1-page = 1-form for unsupervised models.  Otherwise it might be confusing to new people coming to the code.